codegen/src/main/kotlin/org/jetbrains/bsp/generators/ir/IrConfig.kt

```diff
@@ -8,13 +10,35 @@ class IrConfig(
     val untaggedUnions: AbstractionLevel,
 )
 
-enum class AbstractionLevel {
-    AsDef,
-    AsType,
-}
-
+// It specifies how to represent a particular type, if it should be aliased or not.
```

Comment:
ten opis jest spoko, ale bardziej chodziło mi o to co to na poziomie IR zmienia, a nie w wygenerowanym kodzie. W sensie przydałby się przykład [kod w smithy] -> IR. Ale te przykłady co są mogą zostać

Reply:
teraz lepiej?